prow/lib.sh

```diff
@@ -253,17 +257,22 @@ EOF
         done
       done
     }
+
     # Network 1 contains clusters 1, 2 and 3
```

Comment:
I don't understand how we can have a variable number of clusters but then this is fixed?